pkg/api/validation/schema.go

```diff
@@ -66,6 +67,48 @@ func NewSwaggerSchemaFromBytes(data []byte) (Schema, error) {
 	return schema, nil
 }
 
+func (s *SwaggerSchema) validateList(obj map[string]interface{}) error {
+	items, exists := obj["items"]
+	if !exists {
+		return fmt.Errorf("no items field in %#v", obj)
+	}
+	if _, ok := items.([]interface{}); !ok {
+		return fmt.Errorf("items isn't a slice")
+	}
+	for _, item := range items.([]interface{}) {
+		fields, ok := item.(map[string]interface{})
+		if !ok {
+			return fmt.Errorf("item isn't a map[string]interface{} %#v", item)
+		}
+		groupVersion := fields["apiVersion"]
+		if groupVersion == nil {
+			return fmt.Errorf("list item apiVersion not set")
+		}
+		if _, ok := groupVersion.(string); !ok {
+			return fmt.Errorf("apiVersion isn't string type")
```

Comment:
`items[%d].apiVersion isn't string type`, etc, makes it far easier to figure out what to fix